### PR TITLE
Expose find_conda() and get_virtualenv()

### PR DESCRIPTION
**Pull request by javierluraschi**
For users interested in having access to the python executables to interface over command line, exposing the path to the virtualenv and searching for conda installations would certainly be helpful.

### CODE REVIEW

File: R/conda.R [hunk 1]
```diff
@@ -194,7 +194,8 @@ conda_version <- function(conda = "auto") {
 }
 
 
-
+#' @rdname conda-tools
+#' @export
 find_conda <- function() {
```
Comment (jjallaire) [inline R/conda.R]: This function finds the global conda rather than the currently in use conda environment so I don't think it's what you want.

**Comment by jjallaire** [comment]
Let's discuss the actual use case for this PR before we decide whether to merge (there may be a better way)

**Comment by jjallaire** [comment]
This will get you the path to the python binary for the e.g. the "r-mlflow" conda environment:

```r
subset(conda_list(), name == "r-mlflow")$python
```

**Comment by javierluraschi** [comment]
Sounds good. I think you might still want to consider taking the `get_virtualenv()` change to allow others using only virtualenvs to extract this, but since we will scope to conda only support in mlflow, I'll close this one. Thanks!

**Pull request closed without merging**